core/src/main/java/org/apache/iceberg/BaseMetadataTable.java

```diff
@@ -212,6 +212,11 @@ public String toString() {
     return name();
   }
 
+  @Override
+  public int formatVersion() {
+    return table().formatVersion();
```

Comment:
Do metadata tables have format versions? I would argue no?

Reply:
Yeah I don't think it makes sense for metadata tables to have a format version. The entries in the tables could be spread across different "versions" of a table where the underlying table had different format versions.